x-pack/plugins/observability_solution/observability_ai_assistant/server/functions/get_dataset_info/index.ts

```diff
@@ -48,7 +48,7 @@ export function registerGetDatasetInfoFunction({
 
       try {
         const body = await esClient.asCurrentUser.indices.resolveIndex({
-          name: index === '' ? '*' : index.split(','),
+          name: index === '' ? ['*', '*:*'] : index.split(','),
```

Comment:
Will this only query remote indices if index is undefined? Is that the goal or to always query remote clusters?

I'll mention we've had issues query * remote clusters in the past with users wanting to specify which remote clusters because it can take a long time.

Reply:
> Will this only query remote indices if index is not specified? Is that the goal or to always query remote clusters?

the former yes

> I'll mention we've had issues querying * remote clusters in the past with users wanting to specify which remote clusters because it can take a long time if there are many remotes. Perhaps we can have this as a setting at some point.

This only lists the indices, and then the assistant can pick one for querying, so I think that's usually fine. Agreed a setting would be nice (but also a little wary of having too many settings).